README.md

```diff
@@ -1,15 +1,34 @@
-[![GitHub release](https://img.shields.io/github/release/cybozu-go/neco-tenant-controller.svg?maxAge=60)][releases]
-[![CI](https://github.com/cybozu-go/neco-tenant-controller/actions/workflows/ci.yaml/badge.svg)](https://github.com/cybozu-go/neco-tenant-controller/actions/workflows/ci.yaml)
-[![PkgGoDev](https://pkg.go.dev/badge/github.com/cybozu-go/neco-tenant-controller?tab=overview)](https://pkg.go.dev/github.com/cybozu-go/neco-tenant-controller?tab=overview)
-[![Go Report Card](https://goreportcard.com/badge/github.com/cybozu-go/neco-tenant-controller)](https://goreportcard.com/report/github.com/cybozu-go/neco-tenant-controller)
+[![GitHub release](https://img.shields.io/github/release/cybozu-go/cattage.svg?maxAge=60)](https://github.com/cybozu-go/cattage/releases)
+[![CI](https://github.com/cybozu-go/cattage/actions/workflows/ci.yaml/badge.svg)](https://github.com/cybozu-go/cattage/actions/workflows/ci.yaml)
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/cybozu-go/cattage?tab=overview)](https://pkg.go.dev/github.com/cybozu-go/cattage?tab=overview)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cybozu-go/cattage)](https://goreportcard.com/report/github.com/cybozu-go/cattage)
 
-Tenant Controller for Neco
-==========================
+# Cattage
+
+Cattage is a Kubernetes controller that enhances the multi-tenancy of [Argo CD][] with [Accurate][].
 
 **Project Status**: Initial development
 
+## Features
+
+- Management of root-namespaces for tenants. Tenant users will be able to create sub-namespaces in those root-namespaces.
+- When a tenant user creates a sub-namespace, the AppProject will be automatically updated accordingly. Tenant users will be able to deploy applications with Argo CD to the namespaces.
+- The ownership of sub-namespaces can be changed between tenants.
+- Tenant users can create Application resources in their sub-namespaces without `argocd` command. It allows for [App Of Apps Pattern](https://argo-cd.readthedocs.io/en/stable/operator-manual/cluster-bootstrapping/#app-of-apps-pattern) in multi-tenancy environments.
```

Comment:
By using cattage, tenant team members are supposed to deploy Applications on their sub-namespaces and not on the root namespaces. We decided to enforce it in the validation webhook.

Reply:
I'll return a warning message instead of enforcement.
https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/